extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -1709,6 +1709,16 @@
 				value,
 				title;
 
+			if(data.TYPE === "SITEMAP_CHANGED") {
```

Comment:
Nothing :) Grunt expects this format or it complains and does not generate the minified version of the file